docker-build/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+DOCKER_BUILD_DIR="$(dirname -- "$( readlink -f -- "$0"; )";)"
+ENGINE_GIT_DIR="$(dirname "${DOCKER_BUILD_DIR}")"
+CACHE_DIR="${DOCKER_BUILD_DIR}/cache"
+LINUX_LIBS_DIR="${CACHE_DIR}/spring-static-libs"
+WINDOWS_LIBS_DIR="${CACHE_DIR}/mingwlibs64"
+CCACHE_DIR="${CACHE_DIR}/ccache"
+DOCKER_SCRIPTS_DIR="${DOCKER_BUILD_DIR}/scripts"
+
+# Create directories on the host, otherwise docker would create them as root
+mkdir -p "${LINUX_LIBS_DIR}"
+mkdir -p "${WINDOWS_LIBS_DIR}"
+mkdir -p "${CCACHE_DIR}"
+
+docker run -it                                           \
```

Comment:
No -rm? It will create a new container on every run, no?

Reply:
Fixed